metadata-ingestion/src/datahub/cli/docker_cli.py

```diff
@@ -739,7 +736,7 @@ def quickstart(
             check=True,
             env=_docker_subprocess_env(),
         )
-        click.secho("Finished building docker images!")
+        logger.info("Finished building docker images!")
 
     # Start it up! (with retries)
     max_wait_time = datetime.timedelta(minutes=6)
```

Comment:
Any chance we should bump this wait interval? Or make it configurable. I feel like sometimes maybe we just need to give it more time?

Reply:
sure i can bump it to 8

after some point it's just not going to start and any additional time is just adding to user frustration for no reason

Reply:
true true. I'm think in CI it might account for some of what we're seeing